gammapy/data/data_store.py

```diff
@@ -24,6 +24,8 @@
     'ObservationList',
 ]
 
+format = "%(levelname)8s %(filename)s:%(lineno)s - %(funcName)s --- %(message)s"
+logging.basicConfig(level=logging.INFO, format=format)
```

Comment:
Python libraries should never configure loggers. That makes it very hard for users to do so and confusing when `import gammapy.data` changes the global logging behaviour.

So if you keep the logging in `check_observations`, please remove those two lines here and instead put them in the script doing the processing (having it in cli scripts like `hap-data-fits-export` is OK).